text/XXX-A-Compilation-of-Short-Interactive-Tutorials-and-How-To-Guides-for-Coq.md

```diff
@@ -0,0 +1,257 @@
+
+To account for the different versions of Coq and of the plugins, the documentation would be indexed on the releases of the Coq Platform similarly to how the reference manual is for Coq.
+
+We wish this documentation effort to be part of the next version of Coq's website similarly to what can be found on [the OCaml website](https://ocaml.org/docs/lists).
+
+---
+## 4. Organization
+
+### 4.1 First steps
+
+To start the project, we have created a repository on GitHub on which users can already contribute, and a [dedicated channel on Zulip](https://coq.zulipchat.com/#narrow/stream/437203-Platform-docs).
+
+We have already worked on a few tutorials as examples, which can be found on [Théo Zimmermann's repository](https://github.com/Zimmi48/platform-docs). These tutorials are already useful for Coq users and have exposed some undesirable features, bugs or omissions both in the features themselves and their current documentation: see, e.g. this pull request about [Search](https://github.com/coq/coq/pull/18983) and this one about [Equations](https://github.com/mattam82/Coq-Equations/pull/604). We received a lot of reviews from experts.
+
+At the moment, as a proof of concept, we use coqdoc to format our documents and the current version of [jsCoq](https://github.com/jscoq/jscoq). The (very preliminary) output can be seen [here](https://www.theozimmermann.net/platform-docs/).
+As mentioned in section 3.3, we aim in the future to use a more versatile and standard format for `html` document generation.
+
+### 4.2 Writing the documentation
+Writing the different (pedagogical) tutorials will be challenging considering the amount of the work there is to do. Thankfully, many functionalities or plugins of Coq already have documentation in a form or in another.
+Part of this work will involve gathering them, as well as sometimes updating them, or making them more accessible.
+Other tutorials will have to be written from scratch. We will organise directed calls for contributions to avoid having multiple overlapping / rival tutorials.
+Among other possibilities, we will be in touch with plugins designers and expert users for the tutorials and how-to guides of their plugins.
+
+As a base to work on, we suggest an [informal list](https://github.com/Zimmi48/platform-docs/blob/main/draft_structure_doc.md) of tutorials and how-to guides that we could have. This list has been established through informal discussions. It will necessarily evolve with the projects, contributions and user feedback. We further plan to go to CUDW and other similar events to discuss the project and the needs of the community.
```

Comment:
That's quite a long laundry list.  It would be interesting to identify the highest priority tutorials that you or others plan to work on first.

Reply:
The list is mostly there to give an idea of what is possible to people that would like to contribute. The ecosystem is vast: people have different priority and strengths, so I don't think it is important to fix priorities. I would rather have people having fun contribute on what they want. 

This being said, my personal proprieties is to finish tutorials for Equations and then to tackle Ltac2 as I am interested in both, I am in close contact with developers of both, and that is a recurrent request.

Reply:
For priorities, I mean identifying what will provide the greatest benefit to users. That may usefully influence volunteers, but of course each makes their own choices.

Reply:
I guess we can but also it is hard to say what are the priorities. There is a wishlist on zulip. I guess we'll see what people want as we go.